ui.apps/src/main/content/jcr_root/apps/guides-components/components/guidessidenavigation/clientlibs/site/js/guidessidenavigation.ts

```diff
@@ -141,10 +172,14 @@ class GuidesNavigation {
         try {
             const navData = JSON.parse(navigationParent.getAttribute("data-cmp-guides-side-nav-list"));
             const selectedPath = navigationParent.getAttribute("data-cmp-guides-side-nav-current-index");
+            const renderSize = navigationParent.getAttribute("data-cmp-guides-side-nav-items-limit") || "1000";
+            const loadMoreText = navigationParent.getAttribute("data-cmp-guides-side-nav-load-more-text") || "load more...";
```

Comment:
shouldn't "load more..." be translated ?

Reply:
We should use i18 here and also create constants above for these.

Reply:
it already is. Pls check the java code, it always returns a default value that goes thru i18n.